packages/modal-ui/src/lib/components/WhatWallet.tsx

```diff
@@ -4,13 +4,15 @@ interface WhatRow {
   title: string;
   description: string;
   icon: string;
+  iconLight: string;
 }
 
-export const WhatWallet: React.FC<WhatRow> = ({ title, description, icon }) => {
+export const WhatWallet: React.FC<WhatRow> = ({ title, description, icon, iconLight }) => {
   return (
     <div className="wallet-what">
       <div className={"icon-side"}>
-        <img src={icon} alt={"icon"} />
+        <img src={icon} className={"display-dark"} alt={"icon"} />
+        <img src={iconLight} className={"display-light"} alt={"icon"} />
```

Comment:
Why are we adding 2 icons here, I thought all we needed to do is change the background color of the icons.

If we need to change more than just the background would you consider adding the svg code inside the file and removing the img tag and after that with css it is easier to play around with colors?

Reply:
For this one, I have decided to move the markup of WhatWallet inside the WalletHome component since it's just some markup which it's not being repeated/used anywhere else, and this simplifies the code a bit so we don't have to pass props around components since there's not much logic going on here.

I have added the SVG code inside the WalletHome component this way it is easier to work with colors in light/dark themes.

The WhatWallet component has been removed.